### DISCUSSION TITLE
关于ssh_login.sh的问题

### DISCUSSION THREAD

**Discussion by vela-security** [linuxdeepin/developer-center#2993] [Features Request & Ideas | 特性请求 & 头脑风暴]
原来我自定义了一下/usr/lib/ssh_login.sh 文件  , 可以自定以自己的功能
但是自从 升级到5.2.11 后 , 修改//usr/lib/ssh_login.sh 好像不生效了？
是路径修改了吗？ sudo find / -name ssh_login.sh  也发现不了;
现在一大堆之前管理的服务器 远程管理打不开,求解决

**Comment by felixonmars** [comment]
现在把这个文件 embed 为 qt 资源了，所以修改需要手动重新编译。

**Comment by vela-security** [comment]
尴尬！我现在一大堆远程管理 在这里 还链接不上 能 做到配置文件和程序分离吗？不然我手动编译 是不是要提前卸载

**Comment by XMuli** [comment]
@z966768  look

**Comment by BLumia** [comment]
如果是 arch 用户的话还可以考虑装 gtk 版终端作为 workaround 😂

**Comment by meiyixiang** [comment]
你好这个问题我已经记录并反馈，希望提供给你可以定制的能力
